plugins/ltac2/tac2core.ml

```diff
@@ -2066,6 +2066,16 @@ let () = add_scope "constr" (fun arg ->
     Tac2entries.ScopeRule (Procq.Symbol.nterm Procq.Constr.constr, act)
   )
 
+  let () = add_scope "lconstr" (fun arg ->
+    let delimiters = List.map (function
+        | SexprRec (_, { v = Some s }, []) -> s
+        | _ -> scope_fail "lconstr" arg)
+        arg
+    in
+    let act e = Tac2quote.of_constr ~delimiters e in
+    Tac2entries.ScopeRule (Procq.Symbol.nterm Procq.Constr.lconstr, act)
+  )
+
```

Comment:
Nitpick: I know that the immediately surrounding code uses a different style convention, but I think it's a bad idea to use parentheses here. I'd rather have a `begin-end` block like for other scopes. The one for constr and open_constr using parentheses is something that slipped through the review of the PR that introduced it.

(Same remark for the other newly introduced scope below.)